platform/linux-generic/pktio/dpdk.c

```diff
@@ -1290,30 +1290,46 @@ static int dpdk_stats_reset(pktio_entry_t *pktio_entry)
 	return 0;
 }
 
-const pktio_if_ops_t dpdk_pktio_ops = {
-	.name = "dpdk",
-	.init_global = dpdk_pktio_init_global,
-	.init_local = dpdk_pktio_init_local,
-	.term = NULL,
+static pktio_ops_module_t dpdk_pktio_ops = {
```

Comment:
Can we find a solution so these structures are kept as const? If I get this correctly, apart from the linked list embedded in the "base class", there is nothing that should be mutable here, right?

Reply:
Yes, I'm thinking how to achieve this. Can it be later patch improvements.